paddle/fluid/operators/mkldnn/activation_mkldnn_op.cc

```diff
@@ -100,6 +100,14 @@ void eltwise_forward(const framework::ExecutionContext &ctx,
 
   y->set_layout(DataLayout::kMKLDNN);
   y->set_format(GetMKLDNNFormat(*dst_memory_p));
+
+  if (algorithm == mkldnn::algorithm::eltwise_tanh) {
```

Comment:
Can we make unit tests and put dump in unit tests? If not UT, the coverage CI fails too.